src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameUserInputComboBox.xaml

```diff
@@ -0,0 +1,158 @@
+﻿<ComboBox x:Class="Microsoft.CodeAnalysis.InlineRename.UI.SmartRename.SmartRenameUserInputComboBox"
+          xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+          xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+          xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+          xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+          xmlns:vsfx="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+          xmlns:vsui="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+          mc:Ignorable="d"
+          GotKeyboardFocus="ComboBox_GotKeyboardFocus"
+          Unloaded="ComboBox_Unloaded"
+          PreviewKeyUp="ComboBox_PreviewKeyUp"
+          SelectionChanged="ComboBox_SelectionChanged"
+          Text="{Binding IdentifierText, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"
+          ItemsSource="{Binding SmartRenameViewModel.SuggestedNames}"
+          StaysOpenOnEdit="True"
+          IsEditable="True"
+          Style="{StaticResource {x:Static vsfx:VsResourceKeys.ComboBoxStyleKey}}">
+    <ComboBox.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="../InlineRenameColors.xaml"/>
```

Comment:
Is this needed?

Reply:
No, good catch! I struggled to apply the VS theme somehow.